script/core/ActivateSemibook.s.sol

```diff
@@ -65,27 +65,27 @@ contract ActivateSemibook is Test2, Deployer {
     The density is the minimal amount of tokens bought per unit of gas spent.
     The heuristic is: The gas cost of executing an order should represent at
     most 1/COVER_FACTOR the value being bought. In other words: multiply the
-    price of gas in tokens (obtained from the price of gas in gwei and the price
-    of tokens in gwei) by COVER_FACTOR to get the density.
+    price of gas in tokens (obtained from the price of gas in mwei and the price
+    of tokens in mwei) by COVER_FACTOR to get the density.
     
     Units:
-       - outbound_in_gwei is in gwei/display token units
+       - outbound_in_mwei is in mwei/display token units
        - COVER_FACTOR is unitless
        - decN is in (base token units)/(display token units)
```

Comment:
What are `base token units` here? Should it just say `token units`?

Reply:
see my other comment abbove, I always struggled with how to explicitly reference "non-display units"